data/scripts.js

```diff
@@ -549,7 +549,7 @@ exports.BattleScripts = {
 			hitResult = this.runEvent('DragOut', target, pokemon, move);
 			if (hitResult) {
 				target.forceSwitchFlag = true;
-			} else if (hitResult === false) {
+			} else if (hitResult === false && move.status === 'Status') {
```

Comment:
I'm confused. Shouldn't this be `move.category`? Someone please correct me if I'm incorrect.